src/id/doujindesu/src/eu/kanade/tachiyomi/extension/id/doujindesu/DoujinDesu.kt

```diff
@@ -368,29 +377,60 @@ class DoujinDesu : ParsedHttpSource(), ConfigurableSource {
 
     override fun mangaDetailsParse(document: Document): SManga {
         val infoElement = document.select("section.metadata").first()!!
+        val AuthorName = when {
+            document.select("section.metadata > table:nth-child(2) > tbody > tr.pages > td:contains(Author) + td:nth-child(2) > a")
+                .isEmpty() -> "Tidak Diketahui"
```

Comment:
If it's empty you should just not set it. Applies to all attributes in this block. This way the app will localize the field.

Reply:
.isEmpty() -> "" right?
Got it

Reply:
I haven't checked if an empty string triggers the default text. Conditionally setting the artist field should be safe. If an empty string works fine you can use that.

Reply:
Because sometimes the Doujin in DouDesu is didn't have a Author name, sometimes only using Group or didn't have any of that, that's why i combine them on Author Section